syntax_and_semantics/literals/range.md

```diff
@@ -1,14 +1,32 @@
 # Range
 
-A [Range](http://crystal-lang.org/api/Range.html) is typically constructed with a range literal:
+A [Range](http://crystal-lang.org/api/Range.html) represents an interval between two values. It is typically constructed with a range literal, consisting of two `..` or three dots:
 
-```crystal
-x..y  # an inclusive range, in mathematics: [x, y]
-x...y # an exclusive range, in mathematics: [x, y)
+* `x..y`: Two dots denote an inclusive range, including `x` and `y` and all values in between (in mathematics: `[x, y]`) .
```

Comment:
"two dots" and "three dots" are already mentioned above and you see them right before the description. I think this can be kept shorter here.
```suggestion
* `x..y`: Denotes an inclusive range, including `x` and `y` and all values in between (in mathematics: `[x, y]`) .
```

Reply:
I disagree. I'd rather use full sentences here.